doc/langref.html.in

```diff
@@ -2180,16 +2180,20 @@ or
       Unlike normal structs, {#syntax#}packed{#endsyntax#} structs have guaranteed in-memory layout:
       </p>
       <ul>
-        <li>Fields remain in the order declared, least to most significant.</li>
+        <li>Fields are arranged in the order declared, from the least to the most significant bits of a backing integer.</li>
         <li>There is no padding between fields.</li>
-        <li>Zig supports arbitrary width {#link|Integers#} and although normally, integers with fewer
-        than 8 bits will still use 1 byte of memory, in packed structs, they use
-        exactly their bit width.
+        <li>The backing integer is subject to the same rules as any integer, including {#link|alignment|Alignment#}, 
+        having a maximum bit count of 65535, and the host endianness.
+        On a big endian system, the first declared field will have the highest memory address.
         </li>
+        <li>The backing integer has the same bit count as the fields' total bit width.</li> 
+        <li>Field access and assignment can be understood as shorthand for bitshifts on the backing integer.</li>
+        <li>Integers will use exactly as many bits as they have. For example, a {#syntax#}u5{#endsyntax#} will use 5 bits of the backing integer.
```

Comment:
`</li>` end tag missing here, which is reported by the CI failure. (I think that's only one).

Reply:
https://github.com/ziglang/zig/pull/21741/commits/d3f1e191239328b0343d831286f9b7eb308aea3f

Reply:
Thanks! bit of a trip to reproduce this error locally.

I had to:
1. Download superhtml and add it to my PATH (still a bit of a linux novice but I just added the executable to ~/bin/ and ran `chmod +x` on the executable)
2. run `zig build langref -Denable-superhtml`